engine/security/rootless.md

```diff
@@ -0,0 +1,278 @@
+---
+description: Run the Docker daemon as a non-root user (Rootless mode)
+keywords: security, namespaces, rootless
+title: Run the Docker daemon as a non-root user (Rootless mode)
+---
+
+Rootless mode allows running the Docker daemon and containers as a non-root
+user, for the sake of mitigating potentail vulnerabilities in the daemon and
+the container runtime.
+
+Rootless mode does not require root privileges even for installation of the
+Docker daemon, as long as [the prerequisites](#prerequiresites) are satisfied.
+
+Rootless mode was introduced in Docker Engine 19.03.
+
+> **Note**:
+> Rootless mode is an experimental feature and has [limitations](#known-limitations).
+
+## How it works
+Rootless mode executes the Docker daemon and containers inside a user namespace.
+This is very similar to [`userns-remap` mode](userns-remap.md), except that
+with `userns-remap` mode, the daemon itself is running with root privileges, whereas in
+rootless mode, both the daemon and the container are running without root privileges.
+
+Rootless mode does not use binaries with SETUID bits or file capabilities,
+except `newuidmap` and `newgidmap`, which are needed to allow multiple
+UIDs/GIDs to be used in the user namespace.
+
+## Prerequisites
+
+- `newuidmap` and `newgidmap` need to be installed on the host. These commands
+  are provided by the `uidmap` package on most distros.
+
+- `/etc/subuid` and `/etc/subgid` should contain at least 65,536 subordinate
+  UIDs/GIDs for the user. In the following example, the user `testuser` has
+  65,536 subordinate UIDs/GIDs (231072-296607).
+
+```console
+$ id -u
+1001
+$ whoami
+testuser
+$ grep ^$(whoami): /etc/subuid
+testuser:231072:65536
+$ grep ^$(whoami): /etc/subgid
+testuser::231072:65536
+```
+
+### Distribution-specific hint
+
+Using Ubuntu kernel is recommended.
```

Comment:
I would suggest just putting this as a > note instead. So, get rid of the H3 and do something like:

```
> Note: Using Ubuntu kernel is recommended. 
```

Reply:
done